app/models/host/managed.rb

```diff
@@ -1016,4 +1020,8 @@ def send_built_notification
   rescue SocketError, Net::SMTPError => e
     Foreman::Logging.exception("Host has been created. Failed to send email", e)
   end
+
+  def remove_ui_notifications
```

Comment:
Is there any way we can have these changes in a concern `Hostext::Notifications` or similar? I'm just worried we keep on adding things to this object. Plus, `Host::Discovered` uses `Hostext` concerns when it needs behavior from `Host::Managed.`